x-pack/legacy/plugins/maps/server/maps_telemetry/telemetry_task.js

```diff
@@ -41,35 +41,28 @@ export function registerMapsTelemetryTask(server) {
       title: 'Maps telemetry fetch task',
       type: TELEMETRY_TASK_TYPE,
       timeout: '1m',
-      numWorkers: 2,
```

Comment:
I like the changes more if this file has just this line of code change. Maps team can own further cleanup.